doc/api/http2.md

```diff
@@ -271,12 +271,17 @@ server.on('stream', (stream, headers) => {
     'content-type': 'text/html',
     ':status': 200
   });
+  stream.on('error', (error) => console.log(error));
```

Comment:
`console.log` -> `console.error`?

Reply:
Or maybe just:
```js
stream.on('error', console.error);
```

Reply:
:+1: Switched to the first suggestion – I’d prefer to give a user-defined function here, since that is a bit closer to what one would actually do in real-world code